drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -4676,10 +4679,10 @@ void RasterizerStorageGLES2::_render_target_allocate(RenderTarget *rt) {
 			return;
 		}
 
-		texture->format = Image::FORMAT_RGBA8;
-		texture->gl_format_cache = GL_RGBA;
+		texture->format = image_format;
+		texture->gl_format_cache = color_format;
 		texture->gl_type_cache = GL_UNSIGNED_BYTE;
-		texture->gl_internal_format_cache = GL_RGBA;
+		texture->gl_internal_format_cache = color_internal_format;
```

Comment:
Maybe that part is what's actually making things work? i.e. you wouldn't have to specify `8`, but you do need to use the proper `GL_RGB` or `GL_RGBA` matching what's defined above?

Reply:
no, tried it just by changing the GL_RGB8 setting, that was what made the difference. The rest I changed just to clean stuff up